### PR TITLE
Fix #798 rebase of pull request #804

### LINKED ISSUE #798: Help with filter control select
How to preload filter control select with custom data or with data from server side?

### PR DESCRIPTION
**Pull request by raku**
https://github.com/wenzhixin/bootstrap-table/pull/804

### CODE REVIEW

**Comment by wenzhixin** [comment]
It seems there are some problem of Code Format in the pull request.

**Comment by raku** [comment]
What are the problems so I can change theme?

File: src/extensions/filter-control/bootstrap-table-filter-control.js [hunk 1]
```diff
@@ -10,8 +10,8 @@
 
     var sprintf = function (str) {
         var args = arguments,
-            flag = true,
-            i = 1;
+                flag = true,
```
Comment (djhvscf) [inline src/extensions/filter-control/bootstrap-table-filter-control.js]: These extra spaces.. The code is not indented correctly.

**Comment by raku** [comment]
please check it now, i use netbeans, maybe that extra spaces.

**Comment by djhvscf** [comment]
Yes, now it is correct! @wenzhixin you can merge this PR :+1:

**Pull request merged**